PypeS/pype.py

```diff
@@ -236,20 +238,18 @@ def Execute(self):
         for scriptNameAndArguments in self.ScriptList:
             self.PrintLog('')
             scriptName = scriptNameAndArguments[0]
-            moduleName = scriptName
             scriptArguments = scriptNameAndArguments[1]
             imported = True
             try:
-                exec('import '+ moduleName)
-            except ImportError, e:
-                self.PrintError(e.message)
+                module = importlib.import_module('vmtk.'+scriptName)
+                moduleName = module.__name__
+                scriptObjectClasses = [x for x in dir(module) if isclass(getattr(module, x))]
+                scriptObjectClassName = scriptObjectClasses[0]
+            except ImportError as e:
+                self.PrintError(str(e))
                 break
-            scriptObjectClassName = ''
-            exec ('scriptObjectClassName =  '+moduleName+'.'+moduleName)
-            moduleScriptObjectClassName = moduleName+'.'+scriptObjectClassName
-            self.PrintLog('Creating ' + scriptObjectClassName + ' instance.')
-            scriptObject = 0
-            exec ('scriptObject = '+moduleScriptObjectClassName+'()')
+            scriptObject = getattr(module, scriptObjectClassName)
+            scriptObject = scriptObject()
```

Comment:
I wanted to highligh this section (and the import section directly above) since this is a major change to the stucture of the code. 

The combination of lines 244-249 and 251-252 decouples the explicit definition in the first line of a vmtkScript between the files name and the name of the class inside. Instead, the scriptObjectClassName is identified by searching attributes with a file and selecting the first class that it comes across, dynamically creating the scriptObject from a call to that scriptObjectClassName